index.html

```diff
@@ -0,0 +1,143 @@
+                    </p>
+                </div>
+            </div>
+
+            <div class="card card-1">
+                <div class="img-card">
+                    <img src="./images/EnglishPremierLeague.webp" alt="premier">
+                </div>
+                <div class="title-card">
+                    <h4>premier league</h4>
+                </div>
+                <div class="description-card">
+                    <p>
+                        FIFA adopts temporary employment and registration rules to address several issues in relation to war in Ukraine
+                    </p>
+                </div>
+            </div>
+
+            <div class="card card-1">
+                <div class="img-card">
+                    <img src="./images/serie.jpg" alt="serie">
+                </div>
+                <div class="title-card">
+                    <h4>serie A</h4>
+                </div>
+                <div class="description-card">
+                    <p>
+                        FIFA adopts temporary employment and registration rules to address several issues in relation to war in Ukraine
+                    </p>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section class="featured" id="Featured">
+    
+    </section>
+
+    <section class="partner" id="Studio">
+        <h3>Our Partner</h3>
+        <div class="border border-dir"></div>
+        <ul class="Partner-logo">
+          <li><img decoding="async" loading="lazy" height="57" width="114" src="https://cloudinary.fifa.com/m/7ecf457666d9fb6c/original/fwc2022_fp_dalian-wanda-co_lbgr.eps?tx=c_fill,x_57,y_28,g_xy_center,q_auto,w_114,h_57" alt="Wanda group" title="Wanda group" class="image_img__JuDY4"></li>
+          <li><img decoding="async" loading="lazy" height="57" width="114" src="https://cloudinary.fifa.com/m/7b71c2f3185bddfb/original/fwc2022_fp_cocacola-co_lbgr.eps?tx=c_fill,x_57,y_28,g_xy_center,q_auto,w_114,h_57" alt="Coca Cola" title="Coca Cola" class="image_img__JuDY4"></li>
+          <li>
+            <img decoding="async" loading="lazy" height="57" width="114" src="https://cloudinary.fifa.com/m/6d3a4cc8e08eeb0f/webimage-fwc2022_fp_qatar-co_lbgr.png?tx=c_fill,g_auto,q_auto,w_114,h_57" alt="Qatar Airways" title="Qatar Airways" class="image_img__JuDY4">
+          </li>
+          <li>
+            <img decoding="async" loading="lazy" height="71" width="94" src="https://digitalhub.fifa.com/transform/fdae2bc1-3678-4471-955d-20e4870b924f/visa_94x71?io=transform:fill,height:71,width:94" alt="VISA" title="VISA" class="image_img__JuDY4">
+          </li>
+          <li>
+            <img decoding="async" loading="lazy" height="57" width="114" src="https://cloudinary.fifa.com/m/6954f7d3fd116343/webimage-fwc2022_fp_hyundai-kia-co_lbgr.png?tx=c_fill,g_auto,q_auto,w_114,h_57" alt="Hyundai" title="Hyundai" class="image_img__JuDY4">
+          </li>
+        </ul>
+      </section>
+ <script src="main.js"></script>
```

Comment:
- [ ] I noticed that your website doesn't have a footer as defined in the original design. Please add it and make it similar to the original design. It is important as developers to pay attention to details :+1: 

#### Home page footer design
![Screenshot from 2022-12-02 21-30-00](https://user-images.githubusercontent.com/6615394/205391101-c4393786-115e-4a8e-93c6-1cfc254d1208.png)

#### About page footer design
![Screenshot from 2022-12-02 21-30-58](https://user-images.githubusercontent.com/6615394/205391345-753c6b0d-bf2a-47a5-bb66-b2e373fe852a.png)